components/deprecated-en.html

```diff
@@ -24,7 +24,7 @@ <h2>Arm's length branding - Markup change - v4.0.28</h2>
 <aside class="gc-nttvs container">
 	<div class="row mrgn-tp-lg">
 		<div class="col-sm-4 col-xs-10 col-sm-push-8 mrgn-bttm-lg">
-			<img class="img-responsive" alt="Department name" src="{{assets}}/../{{site.theme}}/assets/sig-blk-en.png">
+			<img class="img-responsive" alt="Department name" src="../sites/assets/sig-blk-en.png">
```

Comment:
Is this an appropriate way of dealing with image files that are referenced from the GCWeb repo's site assets?

Reply:
If possible, please duplicate that image because it is only used for the template.